index.html

```diff
@@ -2438,12 +2473,12 @@ <h3>
         </p>
 
         <p>
-The value of the <code>@context</code> object member MUST be ignored as this is
-reserved for JSON-LD consumers.
+If a <code>@context</code> object member is found, the consumer MUST raise
```

Comment:
```suggestion
```

redundant normative statement.

Reply:
Yes it's redundant but also true.

Reply:
Only in a world where DID Core redefines what `JSON` means.